src/test/java/web/tests/FlipkartTestCopy.java

```diff
@@ -0,0 +1,144 @@
+package web.tests;
```

Comment:
why are we creating 2 tests classes here?
The test method should be only in FlipkartTest.java